tests/basic/pool_function_test.go

```diff
@@ -186,3 +189,152 @@ var _ = Describe("{PoolExpandWithPXRestart}", func() {
 		})
 	})
 })
+
+var _ = Describe("{PoolExpandResizeInvalidPoolID}", func() {
+
+	BeforeEach(func() {
+		StartTorpedoTest("PoolExpandResizeInvalidPoolID",
+			"Initiate pool expansion using invalid Id", nil, testrailID)
+	})
+
+	AfterEach(func() {
+		EndTorpedoTest()
+	})
+
+	stepLog := "Resize with invalid pool ID"
+	log.InfoD(stepLog)
+	It(stepLog, func() {
+		// pick node to resize
+		nodes := node.GetStorageNodes()
+		nodetoPick := nodes[0]
```

Comment:
There is no necessity to "pick a node", we can use the logic already implemented in the Smoky suite

Reply:
for the test i wanted to pick the pool which does not depends on app thus used GetStorageNodes to get node